quinn-proto/Cargo.toml

```diff
@@ -18,14 +18,20 @@ maintenance = { status = "experimental" }
 
 [features]
 default = ["tls-rustls"]
+# Enable checking of certificate transparency logs
```

Comment:
Checking of certificate transparency logs is an always-available rustls feature; rather, this bakes in in a specific set of logs and configures them by default, similar to how we formerly offered webpki-roots for baked-in root certs.

Reply:
Replaced with "Use Google's list of CT logs to enable certificate transparency checks".